ColorzCore/EAInterpreter.cs

```diff
@@ -130,14 +130,33 @@ public void Interpret()
                     }
                     line.WriteData(myROM);
                 }
+
                 myROM.WriteROM();
+                return true;
             }
             else
             {
                 serr.WriteLine("Errors occurred; no changes written.");
+                return false;
             }
         }
 
+        public bool WriteNocashSymbols(TextWriter output)
+        {
+            if (myParser == null)
```

Comment:
While this check makes sense, it doesn't *feel* right to me. 

Possibly move the intitalization of myParser (line 49+50) to the constructor (after which the check becomes unnecessary)? I don't see any instance where Interpret may be called twice.

Following that line of logic, perhaps make the whole class static and instead provide an `Initialize` method instead of a constructor, but that's a bit outside of the scope of this PR.